starter-repos/app-template/skaffold.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: skaffold/v2beta7
+kind: Config
+metadata:
+  name: app-template
+build:
+  artifacts:
+  - image: gcr.io/jingfangliu-194020/app:v1
+deploy:
+  kustomize:
+    paths:
+    - config/envs/dev
```

Comment:
Does the dev use local machine?

Reply:
yes. that's  what the most skaffold dev user does.